libtraceevent/parse-filter.c

```diff
@@ -1624,9 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
 		case FILTER_TRIVIAL_FALSE:
 			if (filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
 		case FILTER_TRIVIAL_TRUE:
 			if (!filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
```

Comment:
It's a library function so any user (external program) can use it.  Maybe trace-cmd?

But I'm not sure the fall-through is right for the FALSE case since it'll be hit always.